BUSINESS_REQUIREMENTS.md

```diff
@@ -2,26 +2,47 @@
-10. User can review the products only after they received it.
-11. User can get recommendation based on what they see, what they put to wishlist, what they put to cart, and what they ordered (and how frequent they order it).
-12. User can chat the store owner to ask things.
-13. User can change their profile data, shipping addresses, card information, and more.
-14. Optionally, user can request for account deletion (permanently, not soft deletion).
+4. User can see (and search by query, obviously) available and sold out products, for each product containing ID, name, description, price, available stocks, options, product weight, dimension, and every other fields you can think of.
+5. User can put a product to their cart. They can change product quantity, add more items, and remove items from their cart.
+6. User can put a product to their wishlist. Santa Claus can't grant the wishlist because he doesn't exist, I mean, come on! Grow up!
+7. User can use vouchers and coupons. They can preview the total price after those discounts both on cart and on the order preview page. Coupons must not make the total shopping amount to be less than zero.
+8. User can only apply one coupons per transaction (for multiple coupons capability, see PHASE 2).
+9. User can check out their products from the cart, and choose a payment method, shipping method. They can choose which address to send.
+10. During checkout, user can mark the order as a gift for the person receiving it. 
+11. User can see the status of their order/transaction. Valid statuses will be defined on a separate document.
+12. User CANNOT COMPLAIN about their order/transaction, we'll be too lazy to handle this case. So, there will be NO REFUNDS YET.
+13. User can mark their order as complete by stating that they have received it. 
+14. User can review the products only after they received it.
+15. User can give ratings to the product (1-5 star) and the store owner (1-5 star).
+16. User can get recommendation based on their product browsing history, what they put to wishlist, what they put to cart, and what they ordered (and how frequent they order it).
+17. User can chat the store owner to ask things.
+18. User can change their profile data, shipping addresses, card information, and more.
+19. Optionally, user can request for account deletion (permanently, not soft deletion).
+
 
 Now, about merchant:
 
-15. Merchant can register to have their store created on TokoBapak.
-16. Merchant can put, remove and modify their items on their store.
-17. Merchant can chat to the user, and send a follow up message.
-18. Merchant can receive orders, and see the status of the order.
-19. Merchant can see how rich they are, and their selling statistics.
-20. Merchant can request for withdrawal.
+20. Merchant can register to have their store created on TokoBapak.
+21. Merchant can set their "about me", opening and closing hours, the category that they're selling, and where they are based on. 
```

Comment:
Merchant also have their store badge.
eg: Official store, Mall, Star, etc..

Reply:
This should be handled on later stage.